css/intro.css

```diff
@@ -0,0 +1,82 @@
+.input-label-dark {
+  margin-left: 0.5rem;
+}
+
+.input-big {
+  height: 10.25rem;
+}
+
+#renderHeardAbout {
+  margin-bottom: 1rem;
+}
+
+#loading {
+  margin: 39vh 0;
+  text-align: center;
+}
+
+.not-found {
```

Comment:
Repeating styles can be combined together.
Here `.not-found` and `.not-authorized`

```css
.not-found,
.not-authorized
{
//css code
}
```